user_tools/src/spark_rapids_pytools/cloud_api/sp_types.py

```diff
@@ -393,7 +393,11 @@ def process_streams(std_out, std_err):
                 if len(stdout_splits) > 0:
                     std_out_lines = Utils.gen_multiline_str([f'\t| {line}' for line in stdout_splits])
                     stdout_str = f'\n\t<STDOUT>\n{std_out_lines}'
-                cmd_log_str = Utils.gen_joined_str(' ', process_credentials_option(cmd))
+                if isinstance(cmd, list):
+                    cmd_list = cmd
+                else:
+                    cmd_list = cmd.split(' ')
```

Comment:
not sure if this is more concise, nit: `cmd_list = cmd if isinstance(cmd, list) else cmd.split(' ')`

Reply:
Refactored to make it more concise.